apps/api/src/city/dto/createBootcamp.dto.ts

```diff
@@ -0,0 +1,5 @@
+export class CityCreateDto {
```

Comment:
I don't think we need entity with name `createBootcamp` in the repo.

Is there a reason for it ?

Reply:
This is because i merged the code from master. I will delete it.

Reply:
Done.